configs/katello/3.15.yaml

```diff
@@ -80,5 +80,5 @@
         katello-pulpcore-3.15-el7: 0
     build_package_group_source_tag: katello-pulpcore-nightly-el7-build
     external_repos:
-      - centos7-os
-      - centos7-updates
+      - centos-7-server
+      - centos-7-updates
```

Comment:
Today this doesn't exist. Do you intend to rename it?

Reply:
Vim mass update fail, this is actually named `centos-7-server-updates` and should now be reflected.